Bojzen-I-Mitten/Stort-spelprojekt
# Network/sync game time

Some changes to gui.
Syncing MatchLength, MaxPlayers and ServerName.

Test:
- Start to instances of the game or test with a friend.
- In main menu. The caret for the name will be rotated correctly and follow the name properly.
- Change the game time and max players when hosting a game.
- You can change the server name but that will not be indicated anywhere.
- When the other player joins, it will see the set max players in the select te am menu. It will say how many players that can be in each team.
- The match length will be seen when the game starts.
